schema/rpc.fbs

```diff
@@ -84,15 +84,17 @@ table SettingsRequest {}
 table SettingsResponse {
     steam_vr_trackers: SteamVRTrackersSetting;
     filtering: FilteringSettings;
-    vrc_osc: VRCOSCSettings;
+    osc_router: OSCSettings;
+    vrc_osc: OSCSettings;
     model_settings: solarxr_protocol.rpc.settings.ModelSettings;
     tap_detection_settings: TapDetectionSettings;
 }
 
 table ChangeSettingsRequest {
     steam_vr_trackers: SteamVRTrackersSetting;
     filtering: FilteringSettings;
-    vrc_osc: VRCOSCSettings;
+    osc_router: OSCSettings;
```

Comment:
I don't know what the purpose of osc_router is. Please document it.

Additionally, why does it share the same type with OscSettings? If there are vrchat specific settings those would be in the router too, and vice versa. Seems to me we should have two separate tables, for two separate configurations.

Reply:
Having only 1 table for both makes sending and receiving data much easier since they also have the same config files. We just don’t send or receive trackers for OSCRouter

Reply:
Fixed both problems